server/pull_request.go

```diff
@@ -25,6 +25,7 @@ func (s *Server) handlePullRequestEvent(event *PullRequestEvent) {
 	switch event.Action {
 	case "opened":
 		mlog.Info("PR opened", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number))
+		s.createCLAPendingStatus(pr)
 		s.checkCLA(pr)
```

Comment:
Given that `createCLAPendingStatus` should happen before every `checkCLA`, I think it's worth combining them.

Reply:
Agreed, moved.

Reply:
Did you forgot to remove `s.createCLAPendingStatus`?

Reply:
I did :facepalm: